lib/hammer_cli_katello/content_view_purge.rb

```diff
@@ -84,7 +84,8 @@ def execute
     private def old_unused_versions
       return @old_unused_versions if @old_unused_versions
 
-      all_versions = index(:content_view_versions, :content_view_id => options['option_id'])
+      all_versions = index(:content_view_versions, :content_view_id => options['option_id'],
+                                                   :per_page => 100)
```

Comment:
The original issue mentioned a number of content views even greater than 100.  Do you think it would make sense to loop over the content_view_versions `index` call to ensure even 1000 CVVs would get purged?  In the times when I've heard people using the purge command, they had an obscene amount of archived content view versions.